### PR TITLE
Add Wiimote + Classic Controller support

### PR DESCRIPTION
**Pull request by askotx**

### CODE REVIEW

File: src/platform/wii/main.c [hunk 1]
```diff
@@ -8,6 +8,8 @@
 #include <fat.h>
 #include <gccore.h>
 #include <malloc.h>
+#include <ogcsys.h>
```
Comment (endrift) [inline src/platform/wii/main.c]: Is this `ogcsys.h` include needed?

File: src/platform/wii/main.c [hunk 2]
```diff
@@ -195,35 +198,50 @@ int main() {
 #endif
 		PAD_ScanPads();
 		u16 padkeys = PAD_ButtonsHeld(0);
+		WPAD_ScanPads();
+		u32 WiiPad = WPAD_ButtonsHeld(0);
```
Comment (endrift) [inline src/platform/wii/main.c]: mGBA coding style has variable names starting with lowercase.

File: src/platform/wii/main.c [hunk 3]
```diff
@@ -350,23 +368,37 @@ static void _drawEnd(void) {
...
 	PAD_ScanPads();
 	u16 padkeys = PAD_ButtonsHeld(0);
+  
+	WPAD_ScanPads();
+	u32 WiiPad = WPAD_ButtonsHeld(0);;
+	u32 ext = 0;
+  
 	int keys = 0;
-	if (padkeys & PAD_BUTTON_A) {
+	//gba.keySource = &keys;
```
Comment (endrift) [inline src/platform/wii/main.c]: Please remove any commented out code.

File: src/platform/wii/main.c [hunk 4]
```diff
@@ -350,23 +368,37 @@ static void _drawEnd(void) {
 static int _pollInput(void) {
 	PAD_ScanPads();
 	u16 padkeys = PAD_ButtonsHeld(0);
+  
+	WPAD_ScanPads();
+	u32 WiiPad = WPAD_ButtonsHeld(0);;
```
Comment (endrift) [inline src/platform/wii/main.c]: Extra semicolon. Same comment about case.

**Comment by endrift** [comment]
A few more general notes: for the `if` lines, please match up to the `if` with tabs and then align the rest of the line with spaces. Otherwise, this looks pretty good, thanks!

File: src/platform/wii/main.c (hunk 1 above)
Comment (askotx) [inline src/platform/wii/main.c]: Let me compile and test! 
I'll back soon

**Comment by askotx** [comment]
One question, should I delete this pull request and create a new one with cleanups? or how?

**Comment by endrift** [comment]
You can squash commits and then push force to clean up the existing pull request. There are probably tutorials for this online.

**Comment by askotx** [comment]
Is it ok now? I only get issues with tab/space, my file editor stinks!!

**Comment by endrift** [comment]
Close enough. I can fix it up after the fact.

**Pull request merged**